price-estimator/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:stretch-slim
```

Comment:
If we really want to have an optimized (and tiny) image, maybe we should use `alpine` linux and build `price-estimator` for `musl`?

Just neat a suggestion, not actually something we need to do ATM.

Reply:
Agree that we could do it and agree that we don't need this right now. Can keep in mind for the future.